src/Akka.Persistence.MongoDb/Journal/MongoDbJournal.cs

```diff
@@ -81,48 +94,68 @@ protected override void PreStart()
 
                 if (_settings.AutoInitialize)
                 {
-                    var modelForEntryAndSequenceNr = new CreateIndexModel<JournalEntry>(Builders<JournalEntry>
-                        .IndexKeys
-                        .Ascending(entry => entry.PersistenceId)
-                        .Descending(entry => entry.SequenceNr));
+                    var (perCallCts, unitedCts) = CreatePerCalLCts();
+                    using (perCallCts)
+                    using (unitedCts)
+                    {
+                        var modelForEntryAndSequenceNr = new CreateIndexModel<JournalEntry>(Builders<JournalEntry>
+                            .IndexKeys
+                            .Ascending(entry => entry.PersistenceId)
+                            .Descending(entry => entry.SequenceNr));
 
-                    collection.Indexes
-                        .CreateOneAsync(modelForEntryAndSequenceNr, cancellationToken: CancellationToken.None)
-                        .Wait();
+                        collection.Indexes
+                            .CreateOneAsync(modelForEntryAndSequenceNr, cancellationToken: unitedCts.Token)
```

Comment:
Always use the `unitedCts` on method calls - so if either the `call-timeout` OR the `_pendingCommandsCancellation`, the MongoDb call will still be aborted if the driver supports internally.